### PR TITLE
Add a post about cross-government Slack

### PR DESCRIPTION
**Pull request by stevenjmesser**
While we wait for CDDO to create a page about cross-government Slack, I figured I'd create one so that we can point people to it whenever we get questions through GOV.‌UK Design System support. 

It's a live document that I'll add to over time, and I'm grateful for any suggestions of content that we should add straight away!

### CODE REVIEW

**Comment by frankieroberto** [comment]
(Alternatively we could add something to the https://x-govuk.github.io homepage about joining the x-gov Slack - that's more of a list of resources)

File: app/posts/how-to-use-cross-government-slack.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  src: 
+  alt: 
+  opengraphImage: true
+---
+
+[Cross-government Slack](https://ukgovernmentdigital.slack.com/) is a Slack workspace for anyone working in central government, with channels on a range of topics. It's a place to connect with other people working on digital services across government.
+
+## How to join
+
+The workspace is open to anyone working in central government with a `.gov.uk` email address. If you're a contracter or agency working with a central government organisation, you may need to ask for a `.gov.uk` address to join the Slack.
```
Comment (joelgsamuel) [inline app/posts/how-to-use-cross-government-slack.md:20]: The workspace is open to anyone working in government with a `.gov.uk` email address, and other qualified public sector organisations. If you're a contractor or commercial supplier working with a public sector organisation, you will need to ask for a `.gov.uk` address (or other qualified public sector email address) before you can join this Slack Workspace.

A number of public sector email domains are able to sign-up without being invited, the list of these can be viewed via https://ukgovernmentdigital.slack.com/signup#/domain-signup.

For public sector organisations not on the email domain list, they can be invited by any existing Slack Workspace member.

Comment (frankieroberto) [reply app/posts/how-to-use-cross-government-slack.md:20]: @joelgsamuel thanks! Do you know anything about the history of the cross-gov Slack that you could contribute to the post? Would be nice to set out a bit of the context, and show how it has evolved over time!

Comment (joelgsamuel) [reply app/posts/how-to-use-cross-government-slack.md:20]: Being cautious, I suspect that might need to be reviewed by the CDDO or CO Press Office, so not something I'm comfortable drafting. I'll check in with some folks though!

**Comment by stevenjmesser** [comment]
I've added some more content after @frankieroberto chased me (very kindly!) on x-gov Slack.

Thank you to him and @joelgsamuel for their comments on this PR, it's been helpful.

File: app/posts/how-to-use-cross-government-slack.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: How to use cross-government Slack
+description: Tips on how to join and make the most of the cross-government Slack workspace.
+date: 2023-06-27
+tags: post
+layout: post
```
Comment (paulrobertlloyd) [inline app/posts/how-to-use-cross-government-slack.md:6]: After rebasing `main`, these front matter values are no longer needed.

```suggestion
```

File: app/posts/how-to-use-cross-government-slack.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+author:
+  name: Steve Messer
+  url: https://github.com/stevenjmesser
+image:
+  src: 
+  alt:
```
Comment (paulrobertlloyd) [inline app/posts/how-to-use-cross-government-slack.md:12]: Happy to add an illustration to this post; does anyone know how to add commits to a branch on a forked repo? Alternatively, can merge this in and add illustration later?

File: app/posts/how-to-use-cross-government-slack.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+- Network with other people in similar roles, for sharing techniques and tools
+- Find out about new programmes of work or job vacancies
+- Learn from other teams on how they approach accessibility
+- Discuss and share examples of applying the GOV.UK Design System to your service
+- Get help and support using GOV.UK Pay and GOV.UK Notify
+- Find meeting rooms in other government buildings
+- Keep up with what other people are working on in the [#weeknotes](https://ukgovernmentdigital.slack.com/archives/CE2K1LU3Y) channel
+
+The cool thing about cross-government Slack is that no question is too silly, and there are always kind people around who are willing to help you out. It's an open, inviting community and questions are usually answered quite quickly. I was involved in a workshop looking at the benefits of cross-government Slack recently, and other people who were part of the session said the same. They mentioned shared problem-solving, learning about agile and delivery management, and collaboration as examples of how using cross-government Slack had impacted their day-to-day work positively.
```
Comment (paulrobertlloyd) [inline app/posts/how-to-use-cross-government-slack.md:38]: Break-up paragraph to make it slightly easier to read:

```suggestion
The cool thing about cross-government Slack is that no question is too silly, and there are always kind people around who are willing to help you out. It's an open, inviting community and questions are usually answered quite quickly.

I was involved in a workshop looking at the benefits of cross-government Slack recently, and other people who were part of the session said the same. They mentioned shared problem-solving, learning about agile and delivery management, and collaboration as examples of how using cross-government Slack had impacted their day-to-day work positively.
```

File: app/posts/how-to-use-cross-government-slack.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+author:
+  name: Steve Messer
+  url: https://github.com/stevenjmesser
+image:
+  src: 
+  alt: 
+  opengraphImage: true
+---
+
+[Cross-government Slack](https://ukgovernmentdigital.slack.com/) is a Slack workspace for anyone working in central government, with channels on a range of topics. It's a place to connect with other people working on digital services across government.
```
Comment (paulrobertlloyd) [inline app/posts/how-to-use-cross-government-slack.md:16]: (Me being overly finicky, part 1) Although I think quotes render correctly regardless of input text, think we use smart quotes in source copy (just checked and that’s mostly the case, will go and fix the ones in previous posts that were missed!)

This is the first of a few ‘straight’ quotes:

```suggestion
[Cross-government Slack](https://ukgovernmentdigital.slack.com/) is a Slack workspace for anyone working in central government, with channels on a range of topics. It’s a place to connect with other people working on digital services across government.
```

File: app/posts/how-to-use-cross-government-slack.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+There's tons of reasons to use cross-government Slack. Here's a list of what I've found useful, but there's probably parts of the workspace I haven't made use of yet!
+
+- Network with other people in similar roles, for sharing techniques and tools
+- Find out about new programmes of work or job vacancies
+- Learn from other teams on how they approach accessibility
+- Discuss and share examples of applying the GOV.UK Design System to your service
+- Get help and support using GOV.UK Pay and GOV.UK Notify
+- Find meeting rooms in other government buildings
+- Keep up with what other people are working on in the [#weeknotes](https://ukgovernmentdigital.slack.com/archives/CE2K1LU3Y) channel
```
Comment (paulrobertlloyd) [inline app/posts/how-to-use-cross-government-slack.md:36]: (Me being overly finicky, part 2) Just to be consistent with Markdown used in other posts:

```suggestion
* Network with other people in similar roles, for sharing techniques and tools
* Find out about new programmes of work or job vacancies
* Learn from other teams on how they approach accessibility
* Discuss and share examples of applying the GOV.UK Design System to your service
* Get help and support using GOV.UK Pay and GOV.UK Notify
* Find meeting rooms in other government buildings
* Keep up with what other people are working on in the [#weeknotes](https://ukgovernmentdigital.slack.com/archives/CE2K1LU3Y) channel
```

File: app/posts/how-to-use-cross-government-slack.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+## How to join
+
+The workspace is open to anyone working in government with a `.gov.uk` email address, and other qualified public sector organisations. If you're a contractor or commercial supplier working with a public sector organisation, you will need to ask for a `.gov.uk` address (or other qualified public sector email address) before you can join this Slack workspace.
+
+A number of public sector email domains are able to sign-up without being invited, so check out the [list of allowed email domains](https://ukgovernmentdigital.slack.com/signup#/domain-signup).
+
+For public sector organisations not on the email domain list, they can be invited by any existing Slack workspace member.
```
Comment (paulrobertlloyd) [inline app/posts/how-to-use-cross-government-slack.md:50]: Trailing new line on Markdown posts:

```suggestion
For public sector organisations not on the email domain list, they can be invited by any existing Slack workspace member.

```

**Review by paulrobertlloyd — CHANGES_REQUESTED** [review]
Love this, and great to have another contributor to the site!

Just a few finicky points. Not sure how best to add an illustration to this PR… adding commits to forked branches is something that’s always confused me!

**Comment by frankieroberto** [comment]
> Love this, and great to have another contributor to the site!
> 
> Just a few finicky points. Not sure how best to add an illustration to this PR… adding commits to forked branches is something that’s always confused me!

@paulrobertlloyd I think you can commit directly to the `patch-1` branch on Steven's fork (https://github.com/stevenjmesser/x-govuk.github.io/tree/patch-1) as the "Maintainers are allowed to edit this pull request" option has been selected.

Failing that, open a separate PR to add the image first, and then we can merge that and add reference the image in this PR?

File: app/posts/how-to-use-cross-government-slack.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+There's tons of reasons to use cross-government Slack. Here's a list of what I've found useful, but there's probably parts of the workspace I haven't made use of yet!
+
+* Network with other people in similar roles, for sharing techniques and tools
+* Find out about new programmes of work or job vacancies
+* Learn from other teams on how they approach accessibility
+* Discuss and share examples of applying the GOV.UK Design System to your service
+* Get help and support using GOV.UK Pay and GOV.UK Notify
+* Find meeting rooms in other government buildings
+* Keep up with what other people are working on in the [#weeknotes](https://ukgovernmentdigital.slack.com/archives/CE2K1LU3Y) channel
```
Comment (frankieroberto) [inline app/posts/how-to-use-cross-government-slack.md:34]: This list is great! I think it'd read slightly better with a shorter lead-in, and changing the tense.  Also lets us end the lead in with a colon, and have the list be lowercase. The second bit of the lead in could come after the list?

```suggestion
There's tons of reasons to use cross-government Slack. Here's what I've found useful:

* networking with other people in similar roles, to share techniques and tools
* finding out about new programmes of work or job vacancies
* learning from other teams on how they approach accessibility
* discussing and sharing examples of applying the GOV.UK Design System to services
* getting help and support using GOV.UK Pay and GOV.UK Notify
* finding meeting rooms in other government buildings
* keeping up with what other people are working on in the [#weeknotes](https://ukgovernmentdigital.slack.com/archives/CE2K1LU3Y) channel

There's probably also parts of the workspace I haven't made use of yet!
```

File: app/posts/how-to-use-cross-government-slack.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+The cool thing about cross-government Slack is that no question is too silly, and there are always kind people around who are willing to help you out. It's an open, inviting community and questions are usually answered quite quickly.
+
+I was involved in a workshop looking at the benefits of cross-government Slack recently, and other people who were part of the session said the same. They mentioned shared problem-solving, learning about agile and delivery management, and collaboration as examples of how using cross-government Slack had impacted their day-to-day work positively.
```
Comment (frankieroberto) [inline app/posts/how-to-use-cross-government-slack.md:38]: Suggested rewording, as "said the same" threw me on first read through (presumably a reference to previous paragraph?).

```suggestion
I was involved in a workshop looking at the benefits of cross-government Slack recently with colleagues from across government. We all agreed that the cross-government Slack has positively impacted our day-to-day work by enabling collaboration, shared problem-solving and learning about agile and delivery management.
```

File: app/posts/how-to-use-cross-government-slack.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+[Cross-government Slack](https://ukgovernmentdigital.slack.com/) is a Slack workspace for anyone working in central government, with channels on a range of topics. It’s a place to connect with other people working on digital services across government.
+
+## A bit of history
+
+The cross-government Slack was created by the [Government Digital Service](https://www.gov.uk/government/organisations/government-digital-service) (GDS) in 2015 as a way for GDS to collaborate with other public sector digital service teams.
+
+Over time, the workspace naturally grew and became a place to share knowledge on all things about digital, data and technology in the public sector.
+
+The workspace is overseen by the [Central Digital & Data Office](https://www.gov.uk/government/organisations/central-digital-and-data-office).
```
Comment (frankieroberto) [inline app/posts/how-to-use-cross-government-slack.md:22]:
```suggestion
The workspace is now managed by the [Central Digital & Data Office](https://www.gov.uk/government/organisations/central-digital-and-data-office).
```

File: app/posts/how-to-use-cross-government-slack.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+  opengraphImage: true
+---
+
+[Cross-government Slack](https://ukgovernmentdigital.slack.com/) is a Slack workspace for anyone working in central government, with channels on a range of topics. It’s a place to connect with other people working on digital services across government.
+
+## A bit of history
+
+The cross-government Slack was created by the [Government Digital Service](https://www.gov.uk/government/organisations/government-digital-service) (GDS) in 2015 as a way for GDS to collaborate with other public sector digital service teams.
+
+Over time, the workspace naturally grew and became a place to share knowledge on all things about digital, data and technology in the public sector.
```
Comment (frankieroberto) [inline app/posts/how-to-use-cross-government-slack.md:20]: Worth mentioning that the Slack also started to be used as an official support channel?

```suggestion
Over time, the workspace naturally grew and became a place to share knowledge on all things about digital, data and technology in the public sector. It also started to be used an an official support channel by some product teams, such as the GOV.UK Design System and GOV.UK Notify.
```

**Review by frankieroberto — COMMENTED** [review]
@stevenjmesser thanks for writing this - it's great. Left some minor suggestions.

File: app/posts/how-to-use-cross-government-slack.md (hunk 8 above)
Comment (stevenjmesser) [reply app/posts/how-to-use-cross-government-slack.md:34]: Looks good. Thanks for fixing the Markdown syntax too, I usually use hyphens for lists but didn’t check what was used on other files.

Comment (frankieroberto) [reply app/posts/how-to-use-cross-government-slack.md]: That was @paulrobertlloyd. I don’t really care about such things. 😉

**Review by paulrobertlloyd — CHANGES_REQUESTED** [review]
Added illustration and updated publication date

**Review by paulrobertlloyd — APPROVED** [review]

**Pull request merged**